src/components/SwapForm.tsx

```diff
@@ -294,6 +296,13 @@ const SwapForm = ({
               bordered={false}
               disabled
             />
+            {estimatedMinWithdrawAmount && (
```

Comment:
Be careful with short circuit operators. Maybe we should explicitly cast the expression like this `{!!condition && <Component />}`. We had issues with some of these code snippets displaying `0` if the condition was not met.
If `estimatedMinWithdrawAmount` can be casted to `string` or `number` this might happen in this case as well.

refer to: https://stackoverflow.com/a/53048160